projects/netgrif-components-core/src/lib/data-fields/file-field/abstract-file-field.component.ts

```diff
@@ -5,7 +5,8 @@ import {
 } from '@angular/core';
 import {AbstractDataFieldComponent} from '../models/abstract-data-field-component';
 import {NAE_INFORM_ABOUT_INVALID_DATA} from '../models/invalid-data-policy-token';
-import {FileField} from "./models/file-field";
+import {SafeUrl} from '@angular/platform-browser';
+import {BehaviorSubject} from 'rxjs';
```

Comment:
new imports? They are not used in the file

Reply:
Done